content/guides/models/track/log/customize-logging-axes.md

```diff
@@ -6,51 +6,90 @@ menu:
 title: Customize log axes
 ---
 
-Use `define_metric` to set a **custom x axis**.Custom x-axes are useful in contexts where you need to log to different time steps in the past during training, asynchronously. For example, this can be useful in RL where you may track the per-episode reward and a per-step reward.
+Set a custom x-axis when you log metrics to W&B. By default, W&B logs metrics as *steps*. Each step corresponds to a `wandb.log()` API call. 
 
-[Try `define_metric` live in Google Colab →](http://wandb.me/define-metric-colab)
+For example, suppose you have a `for` loop that loops 10 times (see the following code block). In each `for` loop, you log a metric called `validation_loss` with `wandb.log()`. W&B increments the step number by 1 each time you log that metric. 
 
-### Customize axes
+```python
+import wandb
+
+with wandb.init() as run:
+  # range function creates a sequence of numbers from 0 to 9
+  for i in range(10):
+    log_dict = {
+        "validation_loss": 1/(i+1)   
+    }
+    run.log(log_dict)
+```
+
+If you navigate to your project's workspace, you can see that the `validation_loss` metric is plotted against the step x-axis. Each step in the for loop monotonically increases by 1, so the x-axis shows the step numbers 0, 1, 2, ..., 9. (The Python `range()` function creates a sequence of numbers from 0 to 9.)
 
-By default, all metrics are logged against the same x-axis, which is the W&B internal `step`. Sometimes, you might want to log to a previous step, or use a different x-axis.
+{{< img src="/images/experiments/standard_axes.png" alt="Line plot panel that uses default x-axis. Each step in monotonically increases by 1." >}}
 
-Here's an example of setting a custom x-axis metric, instead of the default step.
+In certain situations, it makes more sense to log metrics against a different x-axis. For example, you may want a logarithmic x-axis. Use the [`define_metric()`]({{< relref "ref/python/run/#define_metric" >}}) method to define a custom x-axis based on a metric you log.
```

Comment:
```suggestion
In certain situations, it makes more sense to log metrics against a different x-axis such as a logarithmic x-axis. Use the [`define_metric()`]({{< relref "ref/python/run/#define_metric" >}}) method to use any metric you log  as a custom x-axis.
```
I got confused between line 29 and 31. Line 29 talks about custom X-axis, then line 31 begins by talking about a custom Y-axis before returning to the custom X-axis. Did something get mixed up along the way?

Reply:
Funny. So that y-value formula was there to make the plots look nice. I can see how this might be confusing. I updated it , let me know if it makes more sense.